blog/2022/05-10-resoto-graph-traversal/index.md

```diff
@@ -0,0 +1,119 @@
+---
+authors: [matthias]
+tags: [graph, search, digitalocean]
+---
+
+# A walk in the graph
+
+Resoto uses a [directed graph](https://en.wikipedia.org/wiki/Directed_graph) to represent your infrastructure resources as nodes and relationships between them as edges. It stores that resource data in a graph database, allowing you to search the graph efficiently. Why should I care, you ask? To answer this question, I would like to present an example that demonstrates the power of this approach.
+
+The following diagram shows an excerpt of the graph model for DigitalOcean droplets. Droplets are the compute instances in DigitalOcean, similar to EC2 instances in AWS. The full model is available in our [reference section](/docs/reference/data-models/digitalocean). Please note that this example uses DigitalOcean, but the same also works on [AWS](/docs/reference/data-models/aws) or [Google Cloud](/docs/reference/data-models/gcp).
```

Comment:
Please note that this example uses DigitalOcean, but the same also works on

--->

Note this example uses DigitalOcean, but works the same for AWS and GCP.